ext/drx/drx.c

```diff
@@ -85,8 +85,15 @@ enum {
     DRX_NOTE_AFLAGS_RESTORE_END,
     DRX_NOTE_COUNT,
 };
-static ptr_uint_t note_base;
-#define NOTE_VAL(enum_val) ((void *)(ptr_int_t)(note_base + (enum_val)))
+
+static ptr_uint_t note_base = 0;
+
+static void *
+get_note_val(int val)
+{
+    ASSERT(note_base > 0, "note_base not initialized");
```

Comment:
0 might be a valid value.  I would again move such an assert to the high-level exported functions if the goal is to detect a user who didn't call drx_init at the right time.

Reply:
Removed this check.